tests/csrf_tests/views.py

```diff
@@ -28,3 +29,8 @@ def non_token_view_using_request_processor(request):
     context = RequestContext(request, processors=[csrf])
     template = Template('')
     return HttpResponse(template.render(context))
+
+
+def error_handler(request, **args):
```

Comment:
I'd use a more specific name and/or docstring that describes what's special about this error handler to trigger the regression since I could imagine future tests may want to use error handlers that have different features. Similar for the `urls.py` and test naming and/or docstring.